bookmark-service/src/main/java/fr/loicmathieu/bookmarkit/Bookmark.java

```diff
@@ -1,7 +1,11 @@
 package fr.loicmathieu.bookmarkit;
 
-public class Bookmark {
-    public Long id;//TODO manage Hibernate ID auto-generation
```

Comment:
I know that my comment is mis-leading but it's the user that should update this entity to Panache.
So maybe just remove the `Long id` and let the user add the `extends PanacheEntity`